src/dashboard/src/Layout/NotificationBox.tsx

```diff
@@ -30,29 +31,58 @@ const usePaperStyle = makeStyles(theme => createStyles({
   }
 }))
 
-const NotificationBox: FunctionComponent<BoxProps> = (props) => {
-  const { notifications } = useContext(ConfigContext)
-  const { currentTeamId } = useContext(TeamContext)
-
-  const notification = useMemo(() => {
-    const notification = get(notifications, [currentTeamId])
-    if (notification === undefined) {
-      return get(notifications, ['.default'])
-    }
-    return notification
-  }, [notifications, currentTeamId])
+const ClusterNotificationBox: FunctionComponent<{ cluster: any, setAmlLink: any}> = ({ cluster, setAmlLink }) => {
```

Comment:
Add types to `setAmlLink` since it's not unknown data structure from backend like `cluster`.